v2/pkg/genruntime/base_types.go

```diff
@@ -16,9 +16,11 @@ type ResourceKind string
 
 const (
 	// ResourceKindNormal is a standard ARM resource.
-	ResourceKindNormal = ResourceKind("normal")
+	ResourceKindNormal = ResourceKind("normal") // TODO: Rename this to ResourceGroup?
```

Comment:
Suggest doing this rename sooner rather than later, maybe even in this PR.

Reply:
Though, is it the kind of resource that changes, or the kind of ownership?

Reply:
It's the scope of the resource that this really is talking about. I've updated this in this PR (although it does make this PR quite a bit bigger)